### ISSUE TITLE
Upload to extension registry

### ISSUE THREAD

**Issue by peterflynn** [theproducer/brackets-ftp#12]
As far as I can tell, this extension isn't currently listed either of these places:
- Official extension registry - https://brackets-registry.aboutweb.com/
- Legacy wiki extension list - https://github.com/adobe/brackets/wiki/Brackets-Extensions

Brackets gets a lot of questions about FTP sync workflows, so I think it would help many people if this extension were more discoverable.

**Comment by theproducer** [comment]
I agree, though the current state of the plugin isn't ready for a huge primetime unveil yet as there are a lot of bugs and refinements that need to be made.  I've been swamped the past few months, but soon I should be able to push forth some badly needed improvements and bug fixes and get it added to the relevant extension registries.

**Comment by JamesParkNINJA** [comment]
Looking forward to it!
